### DISCUSSION TITLE
`gitAuthor` not honoured for bitbucket-server setup?

### DISCUSSION THREAD

**Discussion by amandel** [renovatebot/renovate#10635] [Request Help] [answered]
I try to establish renovatebot in a rather large multi project intranet ci/cd pipeline setup. I've just started this and did some `dryRun` tests. As SCM we use _Bitbucket-Server_. My understanding is that to avoid that branches are treated as owned by renovatebot I have to set `gitAuthor` to an unique emails address. Using a dedicated technical user would complicate the setup a lot. Now renovatebot logs, that it would delete branches as orphan. These branches have last commits from the user configured as `username` but with a different Email address.

```
INFO: DRY-RUN: Would delete orphan branch feature/PROJECT-1229 (repository=PROJECT/project)
```

Is there something I'm doing wrong? Should I add details to this question?

"renovateVersion": "25.50.0"

**Comment by rarkins** [comment, marked as answer]
Renovate will clean up any branches which:
- match the branchPrefix
- aren't known/active to Renovate

gitAuthor isn't taken into consideration when identifying branches, only when deciding if a branch is modified or not. 

Therefore feature/ would not work, but feature/renovate- could, for example. But note they branchPrefix needs to be shared amongst all Renovate branches, so "feature" doesn't seem to fit for many of those.

**Reply by amandel** [reply]
Thanks for the details! I had configured:
```js
    branchPrefix: "feature/",
    additionalBranchPrefix: "sbs-renovate-",
```
and changed this to:
```js
    branchPrefix: "feature/sbs-renovate-",
```
now it works as expected - my other `feature/*` branches are left untouched.
